docs/products/kafka/kafka-mirrormaker/howto/integrate-external-kafka-cluster.rst

```diff
@@ -20,4 +20,108 @@ You can define an external Apache Kafka® service integration endpoint using the
 
 5. The external Apache Kafka cluster is now available under the alias defined in the **Endpoint name** parameter
 
-.. note::  Configure the ACLs for both the source and target cluster such that the MirrorMaker 2 service can describe and create topics, as well as produce and consume messages.
\ No newline at end of file
+
+Permissions and configuration when using external integration
+-------------------------------------------------------------
+
+By default Apache Kafka® MirrorMaker2 creates topics for Apache Kafka® MirrorMaker2 internal data and writes messages to those topics. When using external integration to Kafka the integration data defines the connecting user. For Apache Kafka® MirrorMaker2 to be able to create and write the connecting user needs permissions to do so. It is a requirement to add ACLs to the corresponding Kafka cluster for the connecting users.
+
+For external source the read permission is required to the mirrored topics and on external target the write permission is required to the mirrored topics.
+For the internal topics a read and write permissions is required for connecting user.
+
+The set of topics created by Apache Kafka® MirrorMaker2 to source cluster.
+
+.. list-table::
+  :header-rows: 1
+
+  * - Topic
+    - | Replication factor
+      | (default value)
+    - | Partitions
+      | (default value)
+    - | Cleanup policy
+      | (required policy)
+  * - ``__consumer_offsets``
+    - 3
+    - 50
+    - compact
+  * - ``heartbeats``
```

Comment:
I assume, it also replicates `<target cluster alias>.heartbeats`?

Reply:
I'll fire up the env again to check this. I looked the topics that get created to source and target and I don't think I saw this one. But this could be an artifact of how to configure MM2.

Reply:
This topic is created only when heartbeat emit is enabled. I'll add this detail.

Reply:
Hmm..no. The `<source cluster alias>.heartbeats` is the only created and mirrored heartbeat topic.